package.json

```diff
@@ -47,43 +47,49 @@
   },
   "dependencies": {
     "async": "2.6.2",
-    "body-parser": "~1.19.0",
+    "body-parser": "~1.19.1",
     "command-shell-lib": "1.0.0",
-    "express": "~4.16.4",
-    "got": "~11.8.2",
+    "express": "~4.17.2",
+    "got": "~11.8.3",
     "jexl": "2.3.0",
     "jison": "0.4.18",
     "logops": "2.1.2",
-    "moment": "~2.24.0",
-    "moment-timezone": "~0.5.25",
-    "mongodb": "3.6.8",
-    "mongoose": "5.7.7",
-    "mu2": "~0.5.20",
-    "query-string": "6.5.0",
+    "moment": "~2.29.1",
+    "moment-timezone": "~0.5.34",
+    "mongodb": "3.7.1",
+    "mongoose": "5.13.14",
+    "mu2": "~0.5.21",
+    "query-string": "7.1.0",
     "revalidator": "~0.3.1",
-    "underscore": "~1.12.1",
-    "uuid": "~3.3.2"
+    "underscore": "~1.13.2",
+    "uuid": "~8.3.2"
   },
   "devDependencies": {
-    "coveralls": "~3.1.0",
-    "eslint": "~7.5.0",
-    "eslint-config-tamia": "~7.2.5",
-    "eslint-plugin-prettier": "~3.1.4",
-    "husky": "~4.2.5",
-    "lint-staged": "~10.2.11",
-    "prettier": "~2.0.5",
-    "mocha": "8.0.1",
-    "nock": "13.0.3",
+    "@textlint/types": "~12.1.0",
+    "@textlint/ast-node-types": "~12.1.0",
+    "coveralls": "~3.1.1",
+    "eslint": "~8.6.0",
+    "eslint-config-tamia": "~7.2.7",
+    "eslint-plugin-prettier": "~4.0.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.21.5",
```

Comment:
Fixed bf6e09b.  The additional devDependencies are to avoid peer dependency warnings like:

```text
npm WARN no-cliches@0.3.4 requires a peer of eslint-plugin-import@^2.22.1 but none is installed. You must install peer dependencies yourself.
npm WARN no-cliches@0.3.4 requires a peer of eslint-plugin-jsx-a11y@^6.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN no-cliches@0.3.4 requires a peer of eslint-plugin-react@^7.21.5 but none is installed. You must install peer dependencies yourself.
npm WARN no-cliches@0.3.4 requires a peer of eslint-plugin-react-hooks@^4.0.0 but none is installed. You must install peer dependencies yourself.
```

According to the `package-lock.json` - `no-cliches` is a dependency of `write-good` which is used by `textlint-rule-write-good` which has been upgraded to `~2.0.0` in this PR.